autoload/lsp/inlayhints.vim

```diff
@@ -87,15 +87,15 @@ def LspInlayHintsUpdate(bnr: number)
   endif
 
   var lspserver: dict<any> = buf.BufLspServerGet(bnr, 'inlayHint')
```

Comment:
The BufLspServerGet() function already checks whether the LSP server is ready or not.  It returns the LSP server that is ready.  So there is no need to check for the ready flag below.

Reply:
yea, but i was thinking it was missing something, or anyway inlay hint was not ready anyway.